### PR TITLE
typecasting null to Boolean was resulting in a crash in Search.

### PR DESCRIPTION
**Pull request by varungu**

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @varungu, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by varungu** [comment]
CLA signed

**Comment by boxcla** [comment]
Verified that @varungu has just signed the CLA. Thanks, and we look forward to your contribution.

File: box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxFolder.java [hunk 1]
```diff
@@ -118,7 +118,10 @@ public BoxUploadEmail getUploadEmail() {
      * @return true if the folder has collaborations; otherwise false.
      */
     public Boolean getHasCollaborations() {
-        return (Boolean) mProperties.get(FIELD_HAS_COLLABORATIONS);
+        Object hasCollaborators = mProperties.get(FIELD_HAS_COLLABORATIONS);
+        if (hasCollaborators != null)
+            return (Boolean) hasCollaborators;
+        return false;
```
Comment (brianytang) [inline box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxFolder.java]: This should actually return null as that is actually a valid value that the server can return (ie. property not set)

**Pull request closed without merging**